src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/SyndicationContent.cs

```diff
@@ -21,37 +17,21 @@ protected SyndicationContent()
         {
         }
 
-        protected SyndicationContent(SyndicationContent source)
-        {
-            CopyAttributeExtensions(source);
-        }
+        protected SyndicationContent(SyndicationContent source) => CopyAttributeExtensions(source);
 
         public Dictionary<XmlQualifiedName, string> AttributeExtensions
         {
-            get
-            {
-                if (_attributeExtensions == null)
-                {
-                    _attributeExtensions = new Dictionary<XmlQualifiedName, string>();
-                }
-                return _attributeExtensions;
-            }
+            get => _attributeExtensions ?? (_attributeExtensions = new Dictionary<XmlQualifiedName, string>());
         }
 
-        public abstract string Type
-        {
-            get;
-        }
+        public abstract string Type { get; }
 
         public static TextSyndicationContent CreateHtmlContent(string content)
         {
             return new TextSyndicationContent(content, TextSyndicationContentKind.Html);
         }
 
-        public static TextSyndicationContent CreatePlaintextContent(string content)
-        {
-            return new TextSyndicationContent(content);
-        }
+        public static TextSyndicationContent CreatePlaintextContent(string content) => new TextSyndicationContent(content);
 
```

Comment:
You converted CreatePlaintextContent to an expression-bodied function but didn't convert the other CreateXXXX methods . There should be a consistent convention within a class/file.